cmd/kube-apiserver/app/options/options.go

```diff
@@ -153,6 +155,9 @@ func (s *ServerRunOptions) Flags() (fss cliflag.NamedFlagSets) {
 	fs.BoolVar(&s.AllowPrivileged, "allow-privileged", s.AllowPrivileged,
 		"If true, allow privileged containers. [default=false]")
 
+	fs.StringVar(&s.ShowHiddenMetrics, "show-hidden-metrics", s.ShowHiddenMetrics,
```

Comment:
Should we add a new flag set? e.g. "Metrics flags:". I think it makes sense.

Reply:
Also, invite @brancz for review.

Reply:
Yes I think a new flagset is appropriate, as there will be more to come.